### PR TITLE
audio: volume: move volume related config to component config

### PR DESCRIPTION
**Pull request by btian1**
volume related config are missed to move during volume move out from module adapter, move volume kconfig from module adapter folder to component folder.

### CODE REVIEW

**Comment by btian1** [comment]
@kv2019i , do I need fix the code style check patch problem?

**Review by kv2019i — APPROVED** [review]
No need to change code due to checkpatch warnings, better to move as-is.

File: src/audio/Kconfig [hunk 1]
```diff
@@ -711,4 +711,85 @@ config COMP_MODULE_SHARED_LIBRARY_BUILD
 	  Select if you want to build modules as shared objects that can be used to run
 	  pipelines on the host with the testbench or the ALSA plugin.
 
+config COMP_VOLUME
```
Comment (lgirdwood) [inline src/audio/Kconfig:714]: Can we create a Kconfig for each Module i.e. ```src/audio/volume/Kconfig``` and include it from here.

**Review by lgirdwood — COMMENTED** [review]
Content is good, we jsut need to have the Kconfig in the volume directory.

**Review by singalsu — APPROVED** [review]

**Review by kv2019i — CHANGES_REQUESTED** [review]
Wait a sec, @btian1 commit says we move files but patch diff shows only adding lines. So are the kconfig defs still left to module_adapter?

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@btian1 can you check internal CI. Thanks

**Review by dnikodem — APPROVED** [review]

**Review by dbaluta — APPROVED** [review]

**Pull request merged**